### ISSUE TITLE
Installing depencies in conda takes forever

### ISSUE THREAD

**Issue by TinoCoda** [PromtEngineer/localGPT#834]
After cloning localGPT in my computer, I create a virtual environment using conda with the following command `conda create -n localGPT_llama2` and then I activated the VE using `conda activate localGPT_llama2`.
Running `pip install -r requirements.txt` it takes forever to install the depencies. I tried it on two computers it can easily take hours and hours. 
My first question: is someone else experiencing this problem? 
A subsequent issue it the compatibility problems between some libraries.  I ran into the following error after trying to run `python ingest.py --device_type cpu`

C:\Users\Me\AppData\anaconda3\envs\localGPT_llama2\lib\site-packages\langchain\embeddings\huggingface.py", line 152, in __init__
    raise ValueError("Dependencies for InstructorEmbedding not found.") from e
ValueError: Dependencies for InstructorEmbedding not found.

**Comment by sreeragu555** [comment]
For me it took nearly a hour to install all the packages iguess.

I was able to solve this issue by installing the huggingface-hub package.

'''pip install huggingface-hub===0.24.0'''

**Comment by cswilsnetex** [comment]
@TinoCoda did you ever find out why it takes so long to download dependencies?

**Comment by TinoCoda** [comment]
@cswilsnetex no. But it's mainly connected to the version of hugging face that you choose. The one that is defined in the requirement.txt file is not working well. `pip install huggingface-hub===0.24.0` solved the issue for me. But then I ran into other issues...
